webpack.config.js

```diff
@@ -494,7 +491,7 @@ if (isDev || isTest) {
     // from dev server
     config.output.publicPath = '//localhost:3000/';
 } else {
-    (config.devtool = sourceMap), (config.output.publicPath = '/');
+    config.output.publicPath = '/';
```

Comment:
@alisman what does this change do?

Reply:
@inodb this prevents production build from outputting source maps, which we don't really use at the moment.